api.md

```diff
@@ -580,11 +580,33 @@ the same semantics as the `show_chunks` [function](#show_chunks).
 
 |Name|Description|
 |---|---|
-| `table_name` | Hypertable name from which to drop chunks. If not supplied, all hypertables are affected.
+| `table_name` | Hypertable name or continuous aggregate view from which to drop chunks. This is optional, but see below.
 | `schema_name` | Schema name of the hypertable from which to drop chunks. Defaults to `public`.
 | `cascade` | Boolean on whether to `CASCADE` the drop on chunks, therefore removing dependent objects on chunks to be removed. Defaults to `FALSE`.
 | `cascade_to_materializations` | Set to `TRUE` to also remove chunk data from any associated continuous aggregates. Set to `FALSE` to only drop raw chunks (while keeping data in the continuous aggregates). Defaults to `NULL`, which errors if continuous aggregates exist.|
 
+The `table_name` is optional and can be used to remove chunks from all
```

Comment:
Not quite sure what you are trying to say here?  Does adding `table_name` remove chunks from _all_ hypertables in the database?  The argument description suggests it's used to drop chunks on a specific hypertable.  Or do you mean that `table_name` can be used for _any_ hypertable in the database?

Reply:
Or are you saying that leaving out `table_name` will drop chunks, but only under the following circumstances?

Reply:
Ah, no, sorry my fault. You can remove all chunks in a database if you *don't* provide a table name. Will fix the wording.

Reply:
Fixed. The text was moved to a "known issues" section instead, since it is a slightly cumbersome interface.

Reply:
Removed the section.